packages/plugins/src/foundry/utils/index.ts

```diff
@@ -1176,18 +1097,138 @@ export const findFullyQualifiedNameForInitCode = (
  */
 export const findFullyQualifiedNameForAddress = (
```

Comment:
I defined this function in the previous PR but left it empty because I needed the `accountAccesses` to write it